pandas/_libs/tslibs/nattype.pyx

```diff
@@ -547,35 +547,38 @@ class NaTType(_NaT):
         Parameters
         ----------
         tz : str or timezone object, default None
-            Timezone to localize to
+            Timezone to localize to.
         """)
     round = _make_nat_func('round',  # noqa:E128
         """
         Round the Timestamp to the specified resolution.
 
         Parameters
         ----------
-        freq : a freq string indicating the rounding resolution
-        ambiguous : bool, 'NaT', default 'raise'
-            - bool contains flags to determine if time is dst or not (note
-              that this flag is only applicable for ambiguous fall dst dates)
-            - 'NaT' will return NaT for an ambiguous time
-            - 'raise' will raise an AmbiguousTimeError for an ambiguous time
+        freq : str
+            Frequency string indicating the rounding resolution.
+        ambiguous : bool or {'raise', 'NaT'}, default 'raise'
+            The behavior is as follows:
+
+            * bool contains flags to determine if time is dst or not (note
+              that this flag is only applicable for ambiguous fall dst dates).
+            * 'NaT' will return NaT for an ambiguous time.
+            * 'raise' will raise an AmbiguousTimeError for an ambiguous time.
 
             .. versionadded:: 0.24.0
-        nonexistent : 'shift_forward', 'shift_backward, 'NaT', timedelta, \
-default 'raise'
+        nonexistent : {'raise', 'shift_forward', 'shift_backward, 'NaT', \
+timedelta}, default 'raise'
```

Comment:
nitpick, not sure if there is any way around this, but backslashes like these are bothersome when I want to do code-folding.  Is there an alternative that avoids the backslash?

Reply:
I tried a few variations, but this is the only one that preserved the formatting in the docs and kept the line length < 89

example:
```python
>>> test = """"I am a very long string \
... with a line break"""
>>> print(test)
"I am a very long string with a line break
>>> test2 = r"""I am a very long string literal
... with a line break"""
>>> print(test2)
I am a very long string literal
with a line break
>>> test3 = """I am a very long string
... with a line break"""
>>> print(test3)
I am a very long string
with a line break
```

Reply:
No problem, thanks for looking into this

Reply:
Hmm I wasn't aware of the need for this. Does this affect the HTML output at all? Also think it would be good to not have this if possible

Reply:
this is the docs with the line continuation character:
![image](https://user-images.githubusercontent.com/7444140/66144952-c8bbf000-e5d7-11e9-8b65-115b53ad92df.png)

this is without:
![image](https://user-images.githubusercontent.com/7444140/66145629-403e4f00-e5d9-11e9-8b00-50954fbb35f2.png)

so the html definitely treats it as a new line without the line continuation. definitely not an expert on python strings, so if you guys have any other suggestions on having a line break in the file but having it render as one line, id be happy to try them out. AFAIK, this is the only way.

Reply:
OK sounds good. I'm still OK with this as is can always perfect later